### PR TITLE
resolved clang W-undef warnings

### PR DESCRIPTION
**Pull request by rima1881**
added define for METAIO_USE_NAMESPACE. I wasn't sure if it was intended to be ifdef or if so I didn't change it

### CODE REVIEW

File: metaCommand.h [hunk 1]
```diff
@@ -29,6 +29,10 @@
 #include <list>
 #include <map>
 
+#ifndef METAIO_USE_NAMESPACE
```
Comment (dzenanz) [inline metaCommand.h:32]: This file was probably taken from MetaIO, https://github.com/Kitware/MetaIO/blob/master/src/metaCommand.h

Is this file needed here? Maybe look at its history to find some answers.

**Review by bradking — APPROVED** [review]

**Comment by rima1881** [comment]
Hi, could you please close this pull request? It has been approved for a while now.
